ocean_lib/example_config.py

```diff
@@ -22,73 +23,63 @@
 DEFAULT_PROVIDER_URL = "http://172.15.0.4:8030"
 
 config_defaults = {
-    "NETWORK_NAME": "development",
     "METADATA_CACHE_URI": "http://172.15.0.5:5000",
     "PROVIDER_URL": "http://172.15.0.4:8030",
     "DOWNLOADS_PATH": "consume-downloads",
 }
 
 PROVIDER_PER_NETWORK = {
-    "mainnet": "https://v4.provider.mainnet.oceanprotocol.com",
-    "goerli": "https://v4.provider.goerli.oceanprotocol.com",
-    "bsc": "https://v4.provider.bsc.oceanprotocol.com",
-    "polygon": "https://v4.provider.polygon.oceanprotocol.com",
-    "energyweb": "https://v4.provider.energyweb.oceanprotocol.com",
-    "moonriver": "https://v4.provider.moonriver.oceanprotocol.com",
-    "moonbase": "https://v4.provider.moonbase.oceanprotocol.com",
-    "mumbai": "https://v4.provider.mumbai.oceanprotocol.com",
-    "sepolia": "https://v4.provider.oceanprotocol.com",
-    "development": DEFAULT_PROVIDER_URL,
+    1: "https://v4.provider.mainnet.oceanprotocol.com",
+    5: "https://v4.provider.goerli.oceanprotocol.com",
+    56: "https://v4.provider.bsc.oceanprotocol.com",
+    137: "https://v4.provider.polygon.oceanprotocol.com",
+    246: "https://v4.provider.energyweb.oceanprotocol.com",
+    1285: "https://v4.provider.moonriver.oceanprotocol.com",
+    1287: "https://v4.provider.moonbase.oceanprotocol.com",
+    80001: "https://v4.provider.mumbai.oceanprotocol.com",
+    58008: "https://v4.provider.oceanprotocol.com",
+    8996: DEFAULT_PROVIDER_URL,
 }
 
-
-def get_rpc_url(network_name: str) -> str:
-    """Return the RPC URL for a given network."""
-    if network_name == "development":
-        if os.getenv("DEVELOPMENT_RPC_URL"):
-            return os.getenv("DEVELOPMENT_RPC_URL")
-
-        return "http://localhost:8545"
-
-    base_url = None
-
-    if os.getenv(f"{network_name.upper()}_RPC_URL"):
-        base_url = os.getenv(f"{network_name.upper()}_RPC_URL")
-
-    if os.getenv("WEB3_INFURA_PROJECT_ID"):
-        base_url = f"{base_url}{os.getenv('WEB3_INFURA_PROJECT_ID')}"
-
-    if base_url:
-        return base_url
-
-    raise Exception(f"Need to set {network_name.upper()}_RPC_URL env variable.")
+NAME_PER_NETWORK = {
+    1: "mainnet",
```

Comment:
we should get network name from [addresses.json](https://github.com/oceanprotocol/contracts/blob/main/addresses/address.json) , but we can create another ticket for this